SPECS-EXTENDED/osinfo-db/osinfo-db.spec

```diff
@@ -1,24 +1,62 @@
-Summary:        osinfo database files
-Name:           osinfo-db
-Version:        20221130
-Release:        2%{?dist}
-License:        LGPL-2.1-or-later
-Vendor:         Microsoft Corporation
+%define with_mingw 0
+%if 0%{?fedora}
+    %define with_mingw 0%{!?_without_mingw:1}
+%endif
+
+Summary: 	osinfo database files
+Name: 		osinfo-db
+Version: 	20240701
+Release: 	3
+License: 	GPL-2.0-or-later
+Vendor:         Microsoft Corporation             
 Distribution:   Azure Linux
-URL:            https://libosinfo.org/
-Source0:        https://releases.pagure.org/libosinfo/%{name}-%{version}.tar.xz
-BuildRequires:  intltool
-BuildRequires:  osinfo-db-tools
-Requires:       hwdata
-BuildArch:      noarch
+URL: 		http://libosinfo.org/
+
+Source0: 	https://fedorahosted.org/releases/l/i/libosinfo/%{name}-%{version}.tar.xz
+Source1: 	https://fedorahosted.org/releases/l/i/libosinfo/%{name}-%{version}.tar.xz.asc
+BuildRequires: 	intltool
+BuildRequires: 	osinfo-db-tools
+BuildArch: 	noarch
+Requires: 	hwdata
+
+%if %{with_mingw}
+BuildRequires: mingw32-filesystem >= 95
+BuildRequires: mingw64-filesystem >= 95
+%endif
 
 %description
 The osinfo database provides information about operating systems and
 hypervisor platforms to facilitate the automated configuration and
 provisioning of new virtual machines
 
+%if %{with_mingw}
+%package -n mingw32-osinfo-db
+Summary: %{summary}
+
+%description -n mingw32-osinfo-db
+This package provides the osinfo database of information about
+operating systems for use with virtualization provisioning tools
+
+%package -n mingw64-osinfo-db
+Summary: %{summary}
+
+%description -n mingw64-osinfo-db
+This package provides the osinfo database of information about
+operating systems for use with virtualization provisioning tools
+
+%endif
+
 %install
 osinfo-db-import --root %{buildroot} --dir %{_datadir}/osinfo %{SOURCE0}
+%if 0%{?rhel}
+# Remove the upstream virtio-win / spice-guest-tools drivers
+find %{buildroot}/%{_datadir}/osinfo/os/microsoft.com/ -name "win-*.d" -type d -exec rm -rf {} +
+%endif
+
+%if %{with_mingw}
+osinfo-db-import --root %{buildroot} --dir %{mingw32_datadir}/osinfo %{SOURCE0}
+osinfo-db-import --root %{buildroot} --dir %{mingw64_datadir}/osinfo %{SOURCE0}
```

Comment:
No need to use rhel and mingw related macros. Apply this comment to other parts of the spec file wherever needed.